### PR TITLE
moved some ImportVCFSuite tests to Python

### PR DESCRIPTION
**Pull request by cseed**

### CODE REVIEW

File: python/hail/tests/test_io.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                       GT = hl.call(1), AD = [0, 6], GQ = 70),
+             hl.struct(locus = hl.locus("X", 16061250), s = "C1046::HG02024",
+                       GT = hl.call(2, 2), AD = [0, 0, 11], GQ = 33),
+             hl.struct(locus = hl.locus("X", 16061250), s = "C1046::HG02025",
+                       GT = hl.call(2), AD = [0, 0, 9], GQ = 24)],
+            key=['locus', 's'])
+
+        mt = hl.import_vcf(resource('haploid.vcf'))
+        mt = mt.filter_rows((mt.locus.position == 16050036) | (mt.locus.position == 16061250))
+        mt = mt.filter_cols((mt.s == "C1046::HG02024") | (mt.s == "C1046::HG02025"))
```
Comment (jbloom22) [inline python/hail/tests/test_io.py]: These two filters aren't necessary. Are you including them intentionally?

File: src/main/scala/is/hail/expr/Relational.scala [hunk 2]
```diff
@@ -1710,7 +1710,7 @@ case class MatrixEntriesTable(child: MatrixIR) extends TableIR {
   }
 
   val typ: TableType = child.typ.entriesTableType
-
+
```
Comment (jbloom22) [inline src/main/scala/is/hail/expr/Relational.scala]: take out if you make other changes

File: python/hail/tests/test_io.py (hunk 1 above)
Comment (jbloom22) [reply python/hail/tests/test_io.py]: `haploid.vcf` only has four entries to begin with

Comment (cseed) [reply python/hail/tests/test_io.py]: So it does!  Merged, so will make another PR.

**Pull request merged**